.travis.yml

```diff
@@ -1,16 +1,19 @@
 language: php
 
 php:
-  - 5.3
   - 5.4
   - 5.5
   - 5.6
 
-matrix:
-  allow_failures:
-    - php: 5.6
-
 env:
+  global:
```

Comment:
This block(9-16) was deleted by me in #742.
Packages versions should be specified in ansible/es-playbook.yml now.
Actually, travis built your pull request with ES 1.3.4, as i can see from logs :)